examples/src/main/java/kafka/examples/Consumer.java

```diff
@@ -16,52 +16,40 @@
  */
 package kafka.examples;
 
-
-import java.nio.ByteBuffer;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.Collections;
 import java.util.Properties;
-import kafka.consumer.ConsumerConfig;
-import kafka.consumer.KafkaStream;
-import kafka.javaapi.consumer.ConsumerConnector;
-import kafka.message.MessageAndMetadata;
 
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
 
 public class Consumer extends Thread
 {
-  private final ConsumerConnector consumer;
+  private final KafkaConsumer<Integer, String> consumer;
   private final String topic;
   
   public Consumer(String topic)
   {
-    consumer = kafka.consumer.Consumer.createJavaConsumerConnector(
-            createConsumerConfig());
-    this.topic = topic;
-  }
-
-  private static ConsumerConfig createConsumerConfig()
-  {
     Properties props = new Properties();
-    props.put("zookeeper.connect", KafkaProperties.zkConnect);
-    props.put("group.id", KafkaProperties.groupId);
-    props.put("zookeeper.session.timeout.ms", "400");
-    props.put("zookeeper.sync.time.ms", "200");
+    props.put("bootstrap.servers", "localhost:9092");
```

Comment:
Could we use ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG etc instead of hand-coded strings? It is less error-prone for possible future changes.